### PR TITLE
Publish Docker images to GHCR.io and Quay.io

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

This updates our Docker workflow to publish to https://ghcr.io and https://quay.io in parallel with publishing to Docker Hub. It also adds some basic image metadata to our Docker images.

##### Test Plan

Merge it and see what happens.

##### Additional Information

Fixes: #10307 

<details> <summary>For users: How does this change affect me?</summary>
Users will be able to obtain official Netdata docker images from ghcr.io or quay.io.
</details>

### CODE REVIEW

**Review by Dim-P — DISMISSED** [review]

File: packaging/docker/Dockerfile [hunk 1]
```diff
@@ -119,3 +119,11 @@ ENTRYPOINT ["/usr/sbin/run.sh"]
 HEALTHCHECK --interval=60s --timeout=10s --retries=3 CMD /usr/sbin/health.sh
 
 ONBUILD ENV NETDATA_OFFICIAL_IMAGE=false
+
+LABEL org.opencontainers.image.authors="Netdatabot <bot@netdata.cloud>"
+LABEL org.opencontainers.image.url="https://netdata.cloud"
+LABEL org.opencontainers.image.documentation="https://learn.netdata.cloud"
+LABEL org.opencontainers.image.source="https://github.com/netdata/netdata"
+LABEL org.opencontainers.image.title="Netdata Agent"
+LABEL org.opencontainers.image.description="Official Netdata Agent Docker Imaage"
```
Comment (ilyam8) [inline packaging/docker/Dockerfile:128]:
```suggestion
LABEL org.opencontainers.image.description="Official Netdata Agent Docker Image"
```

**Comment by Ferroin** [comment]
Closing and reopening to forcibly retrigger broken CI.

**Review by Dim-P — APPROVED** [review]

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Comment by andrewm4894** [comment]
@Ferroin is there any docs updated needed as part of this?

**Comment by Ferroin** [comment]
> @Ferroin is there any docs updated needed as part of this?
> 
> e.g. would somewhere in here make sense? https://github.com/netdata/netdata/blob/master/packaging/docker/README.md
> 
> (would have a go at a PR myself but not quite sure where or how best to add it)

Not particularly IMO.

Maybe we might want to mention which registries we have the images published on, but I do not see that as urgent.

**Pull request merged**